### PR TITLE
fix: update names of pharia-1 models to lowercase

### PR DESCRIPTION
**Pull request by moldhouse**
this commit aligns with fresh deployments of the api-scheduler expecting model names to be lowercase

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,11 @@
 ### Breaking Changes
 ...
 
+## 7.2.0
+
+### Fixes
+- Update names of `pharia-1` models to lowercase, aligning with fresh deployments of the api-scheduler.
```
Comment (silver472) [inline CHANGELOG.md:18]: Should this be under the `## Unreleased` heading?

Comment (SebastianNiehusAA) [reply CHANGELOG.md:18]: Yes. We put everything under `## Unreleased` until we create the actual release. Please move it up and remove the `## 7.2.0 section`

File: pyproject.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "intelligence-layer"
-version = "7.1.0"
+version = "7.2.0"
```
Comment (SebastianNiehusAA) [inline pyproject.toml:7]: Thank you for thinking about this, but we will do this ourselves when we do the next release

Comment (SebastianNiehusAA) [inline pyproject.toml:7]:
```suggestion
version = "7.1.0"
```

**Review by SebastianNiehusAA — CHANGES_REQUESTED** [review]

**Review by SebastianNiehusAA — APPROVED** [review]

**Pull request merged**